beam/www/beam/types/scan.ts

```diff
@@ -53,7 +53,7 @@ export type ScanContext = {
 }
 
 export type ScanConfig = {
-	client?: Record<string, string[]>[]
+	client?: Record<string, string[]> | never[]
```

Comment:
Note: if no custom `beam_client` hooks exist, the response from the server is always an empty list. So accounting types for that.